### PR TITLE
UI - Proposed message display update

### PR DESCRIPTION
**Pull request by DavidChristiansen**
• Responsive design

### CODE REVIEW

**Comment by DavidChristiansen** [comment]
Here it is in action - http://jabbr.davidchristiansen.com/?#/rooms/Room1

**Comment by davidfowl** [comment]
Not sure I like this change. Don't like the font or the color scheme change for messages. UI changes like this require alot more back and forth in the https://jabbr.net/#/rooms/meta room.

There's more wasted space for the name than in the current version (and we already have wasted space).

**Comment by davidfowl** [comment]
@DavidChristiansen one behavior I want from this design is what happens when you shrink (i.e. names disappearing). Can you do that without the other style changes? Then we can discuss the other changes separately

File: JabbR/Views/Home/index.cshtml [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <link href="@Url.Content("~/Content/images/logo32.png")" rel="icon" type="image/png" sizes="32x32">
     <link href="@Url.Content("~/Content/images/logo64.png")" rel="icon" type="image/png" sizes="64x64">
     <link href="@Url.Content("~/favicon.ico")" rel="shortcut icon" type="image/x-icon" sizes="16x16">
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans:regular,bold|Terminal+Dosis:500" media="screen" rel="stylesheet" type="text/css">
```
Comment (davidfowl) [inline JabbR/Views/Home/index.cshtml]: Do we need this?

File: README.md [hunk 2]
```diff
@@ -60,3 +60,4 @@ Inline image and content support for your favorite sites:
 
 #### And if you ever happen to get lost...
     Type /? - to show the full list of JabbR Commands
+
```
Comment (davidfowl) [inline README.md]: How did this happen?

**Pull request closed without merging**